### PR TITLE
Cut tree

### PR DESCRIPTION
**Pull request by Shotgunosine**
Adding functionality to remove scans and associated files from a bids tree and either put them in another directory or delete them. Also added creation of metadata files to _make_bids_tree.

### CODE REVIEW

**Comment by Shotgunosine** [comment]
Shit, don't approve this PR yet, I forgot about updating the run numbers.

**Comment by Shotgunosine** [comment]
Wait, do we have to update run numbers? It seems like files don't throw errors if the run numbers aren't updated. What do you think @leej3? There is a certain honesty to keeping the run numbers reflecting deleted scans.

File: heudiconv_helpers/test/test_helpers.py [hunk 1]
```diff
@@ -5,20 +5,18 @@
...
 
 from heudiconv_helpers.helpers import (gen_slice_timings, make_heud_call,
                                        host_is_hpc, validate_heuristics_output,
-                                       dry_run_heurs)
-from heudiconv_helpers.helpers import _set_fields
-from heudiconv_helpers.helpers import _get_fields
-from heudiconv_helpers.helpers import _del_fields
-from heudiconv_helpers.helpers import _get_outcmd
-from heudiconv_helpers.helpers import __get_seqinfo
+                                       dry_run_heurs, _set_fields, _get_fields,
```
Comment (leej3) [inline heudiconv_helpers/test/test_helpers.py:17]: how bout just importing as hh and not specifying all of the functions

**Review by leej3 — COMMENTED** [review]
Looks good. yeah I don't think it's worth much effort to try and alter the run numbers. if its trivial to change then a choice would be best.

File: heudiconv_helpers/test/test_helpers.py (hunk 1 above)
Comment (Shotgunosine) [reply heudiconv_helpers/test/test_helpers.py:17]: Sure, we should also standardize the private functions, do you want to use one _ or two?

**Comment by Shotgunosine** [comment]
I think its sufficiently beyond trivial that I'd rather not mess with it at the moment, it would also complicate reversing the removal of the files from the tree.

File: heudiconv_helpers/test/test_helpers.py (hunk 1 above)
Comment (leej3) [reply heudiconv_helpers/test/test_helpers.py:17]: i think 1. apologies if i've been peppering the code with 2.

**Comment by leej3** [comment]
sounds good. good to merge then once the private functions are fixed

**Pull request merged**